app/src/main/java/io/github/gsantner/memetastic/activity/MemeCreateActivity.java

```diff
@@ -513,6 +521,37 @@ public void onClick(View v) {
         });
     }
 
+
+    @Override
+    public void onColorSelected(int id, @ColorInt int i1) {
```

Comment:
I would use the ressource id here as parameter, than you can switch on R.id.something, and dont have to introduce new IDs :)

Reply:
okay